plugins/pixart-rf/fu-pxi-receiver-device.c

```diff
@@ -601,16 +627,19 @@ fu_pxi_receiver_device_write_firmware(FuDevice *device,
 static gboolean
 fu_pxi_receiver_device_get_peripheral_info(FuPxiReceiverDevice *device,
 					   struct ota_fw_dev_model *model,
+					   guint idx,
 					   GError **error)
 {
 	guint8 buf[FU_PXI_RECEIVER_DEVICE_OTA_BUF_SZ] = {0x0};
 	guint16 checksum = 0;
+	guint16 hpac_ver = 0;
 	g_autofree gchar *version_str = NULL;
 	g_autoptr(GByteArray) ota_cmd = g_byte_array_new();
 	g_autoptr(GByteArray) receiver_device_cmd = g_byte_array_new();
 
 	fu_byte_array_append_uint8(ota_cmd, 0x1); /* ota init new command length */
 	fu_byte_array_append_uint8(ota_cmd, FU_PXI_DEVICE_CMD_FW_OTA_GET_MODEL);
+	fu_byte_array_append_uint8(ota_cmd, idx);
```

Comment:
isn't this an API break? i.e. how will older devices know there's an index here?

Reply:
We originally reserved this parameter in firmware, which was not utilized by older devices. Now, the 'idx' parameter is being used, so it doesn't affect OTA for older devices.